src/clear/sql/sql.cr

```diff
@@ -49,13 +49,17 @@ module Clear
     include Clear::SQL::Logger
     extend self
 
-    @@connection : DB::Database?
+    @@connections : Hash(Symbolic, DB::Database?)?
```

Comment:
I think this object can be not nullable.

Also, the `Symbolic` in this case is not good. Actually I even think that's was a mistake I've made, and I should have used only `Symbol` or `String`. We should avoid mixed key-type hashes (something I hate in Ruby). 

```crystal
@@connections = {} of String => DB::Database?
```